.eslintrc

```diff
@@ -22,6 +28,8 @@
     "arrow-parens": 0,
     "jsx-a11y/control-has-associated-label": 0,
     "react/prop-types": 0,
-    "no-multiple-empty-lines": ["error", { "max": 2, "maxBOF": 0, "maxEOF": 0 }]
+    "no-multiple-empty-lines": ["error", { "max": 2, "maxBOF": 0, "maxEOF": 0 }],
+    "no-unused-vars": 0,
```

Comment:
why are these disabled?

Reply:
In some cases ESLInt returns that React is not being used, but it is not.